script.js

```diff
@@ -0,0 +1,99 @@
+//Task 1
+console.log(+(0.1 + 0.2).toFixed(2));
+
+//Task 2
+const userLogin = prompt("Please enter login");
+
+const userPassword = prompt("Please enter password");
+
+const userAdmin = "admin";
+
+const adminPassword = "12pass33210";
+
+userLogin == userAdmin && userPassword == adminPassword
+  ? console.log("succsess login")
+  : userLogin != userAdmin || userPassword !== adminPassword
+  ? console.log("problem with login or password")
+  : console.log("user doesn't have access to the admin panel");
+
+//Task 3
+const ukraineCurrency = "UAH";
+const watermelon = "watermelon";
+const carrot = "carrot";
+const potato = "potato";
+const watermelonPrice = 21;
+const carrotPrice = 8;
+const potatoPrice = 12;
+const UnfortunatlyMessage =
+  "Unfortunately, you do not have enough money to buy at least 1 pc of any products";
+const tooCheap ='Insufficient funds'
+const userBalance = +prompt("How much money do you have?");
+const lowerProductprice = 8;
+console.log(
+  `${watermelon}: ${watermelonPrice} ${ukraineCurrency}, ${carrot}: ${carrotPrice} ${ukraineCurrency}, ${potato}: ${potatoPrice} ${ukraineCurrency}`
+);
+if (lowerProductprice > userBalance) {
+  console.log(UnfortunatlyMessage);
+} else {
+  const userSelection = prompt("What would you like to by??").toLowerCase();
+  if (
+    userSelection !== watermelon &&
+    userSelection !== potato &&
+    userSelection !== carrot
+  ) {
+    console.log("This product not found");
+  } else if (userSelection === watermelon) {
+    if (userBalance >= watermelonPrice) {
+      console.log(
+        `For ${userBalance} ${ukraineCurrency}. You can by ${Math.trunc(
+          userBalance / watermelonPrice
+        )}pc of this product. You will have remainder ${
+          userBalance % watermelonPrice
+        } ${ukraineCurrency}`
+      );
+    } else {
+      console.log(tooCheap);
```

Comment:
`console.log(tooCheap);` is repeating a lot of times in the code

We can adjust this condition, to be called once - in case when we have not enough money to place it at the end of `if` statements (or `switch`)